morhendos/quick-journal-app
# Add user registration with localStorage

- Added localStorage user management with USERS_STORAGE_KEY
- Implemented registerUser function
- Updated signup page to use registerUser
- Added error handling and validation
- Redirects to login with success query param

Testing:
1. Submit registration form with valid data
2. Check localStorage for saved user
3. Try logging in with new credentials
4. Verify duplicate email handling
5. Test password validation
